rest-api/src/main/java/gov/cms/qpp/conversion/api/helper/MetadataHelper.java

```diff
@@ -36,27 +44,57 @@ public static Metadata generateMetadata(Node node, Outcome outcome) {
 		metadata.setApm(findApm(node));
 		metadata.setTin(findTin(node));
 		metadata.setNpi(findNpi(node));
-		metadata.setCpc(isCpc(node));
+
+		metadata.setCpc(isCpc(node) ? cpcHash() : null);
```

Comment:
a thought, if this is the only place `isCpc` is used maybe this `metadata.setCpc(isCpc(node) ? cpcHash() : null);` could be replaced by `metadata.setCpc(deriveCpcHash(node));`